src/mods.cpp

```diff
@@ -393,9 +394,12 @@ bool ModMetadata::load(const std::string &root_path)
 		return false;
 	}
 
-	Json::Reader reader;
 	Json::Value root;
-	if (!reader.parse(is, root)) {
+	Json::CharReaderBuilder builder;
+	builder.settings_["collectComments"] = false;
+	std::string errs;
+
+	if (!Json::parseFromStream(builder, is, &root, &errs)) {
 		errorstream << "ModMetadata[" << m_mod_name << "]: failed read data "
 			"(Json decoding failure)." << std::endl;
```

Comment:
Also write `errs` to the logger, as we've got information about the cause why it failed.